react-native.config.js

```diff
@@ -7,7 +7,10 @@ const project = (() => {
         sourceDir: path.join('example', 'android'),
       },
       ios: {
-        sourceDir: 'example/ios',
+        sourceDir: path.join('example', 'ios'),
+      },
+      macos: {
+        sourceDir: path.join('example', 'maacos'),
```

Comment:
Looks like a typo. Does it work correctly on `macos`?

Reply:
My bad, I'll forward fix & test.

Reply:
Thank you!